tests/main/snap-service-refresh-mode/task.yaml

```diff
@@ -38,9 +46,14 @@ execute: |
 
     echo "But regular services are restarted"
     journalctl -u snap.test-snapd-service.test-snapd-service | MATCH "stop service"
+    systemctl show -p MainPID snap.test-snapd-service.test-snapd-service > new-main.pid
+    test ! "$(cat new-main.pid)" = "$(cat old-main.pid)"
```

Comment:
Nitpick: can you please make this != rather than ! .. = .. please?

Reply:
Ack, will push a commit.